cnf-tests/testsuites/e2esuite/cnftests-junit.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this is here by mistake (I always do the same :P )

Reply:
good catch, thx :-). Fixed!